src/Module/Manifest.php

```diff
@@ -31,18 +31,19 @@ public static function rawContent(array $parameters = [])
 	{
 		$config = DI::config();
 
-		$tpl = Renderer::getMarkupTemplate('manifest.tpl');
-
 		header('Content-type: application/manifest+json');
 
-		$touch_icon = $config->get('system', 'touch_icon', 'images/friendica-128.png');
-		if ($touch_icon == '') {
-			$touch_icon = 'images/friendica-128.png';
-		}
+		$touch_icon = $config->get('system', 'touch_icon') ?: 'images/friendica-128.png';
+
+		$theme = DI::config()->get('system', 'theme');
 
-		$output = Renderer::replaceMacros($tpl, [
-			'$touch_icon' => $touch_icon,
-			'$title' => $config->get('config', 'sitename', 'Friendica'),
+		$tpl = Core\Renderer::getMarkupTemplate('manifest.tpl');
```

Comment:
Please create a native JSON and don't use the template doing so.

Reply:
I did wonder if I should do that, but I didn't want to go into another tangent in an unrelated PR. Will do.

Reply:
There is a good argument doing it complete: You added the site name. This could contain stuff that needs to be encoded - but JSON encodes different than HTML. So this would create ugly data.

Reply:
Done.